index.html

```diff
@@ -12,36 +12,183 @@
+                    <a class="github" href="www.github.com" target="_blank"><img alt="github" width="18px" height="20px" src="images/github.png"></a>
+                </li>
+                <li>
+                    <a class="linkedin" href="www.linkedin.com" target="_blank"><img class="lin" alt="linkedin" width="20px" height="20px" src="images/linkedin.png"></a>
+                </li>
+                <li>
+                    <a class="angelist" href="https://wellfound.com/" rel="noopener noreferrer" target="_blank"><img alt="logo-angelist" width="15px" height="20px" src="images/sign.png"></a>
+                </li>
+                <li>
+                    <a class="twitter" href="www.twitter.com" target="_blank"><img alt="twitter" width="20px" height="20px" src="images/twitter.png"></a>
+                </li>
+                <li>
+                    <a class="m" href="https://medium.com/" rel="noopener noreferrer" target="_blank"><img alt="logo-m" width="20px" height="16px" src="images/m.png"></a>
+                </li>
+            </div>
+        </ul>
+        <nav>
+            <button type="button" class="my-resume">Get my resume</button>
+        </nav>
+        <div class="languages">
+            <p class="lang poppins">Languages</p>
+            <button name="button-arrow1" type="button"><img class="arrow-down" alt="" width="26px" height= "14px" src="images/flecha-abajo.png"></button>
+        </div>
+        <ul class="lang-sub">
+            <div class="lang1">
+                <li class="li-sp">
+                    <div class="row1">
+                        <img class="js-img" alt="JS Logo" width="100px" height= "100px" src="images/js.png">
+                        <p class="js poppins">JavaScript</p>
+                    </div>
+                </li>
+                <li class="li-sp">
+                    <div class="row1">
+                        <img class="html5-img" alt="HTML5 Logo" width="100px" height="101px" src="images/html5.png">
+                        <p class="html5 poppins">HTML</p>
+                    </div>
+                </li>
+                <li class="li-sp">
+                    <div class="row1">
+                        <img class="css3-img" alt="CSS3 Logo" width="100px" height="101px" src="images/css3.png">
+                        <p class="css3 poppins">CSS</p>
+                    </div>
+                </li>
+            </div>
+        </ul>
+        </div>
+        <hr/>
+        <div class="frameworks">
+                <p class="frame poppins">Frameworks</p>
+                <button name="button-arrow2" type="button"><img class="arrow-right" alt="" width="15px" height= "26px" src="images/flecha-der.png"></button>
+        </div>
+        <hr/>
+        <div class="skills">
+                <p class="skill poppins">Skills</p>
+                <button name="button-arrow3" type="button"> <img class="arrow-right" alt="" width="15px" height= "26px" src="images/flecha-der.png"></button>
```

Comment:
- [ ] You have done an excellent job implementing the required changes :+1:. However, as pointed out by the previous reviewer, please note that the arrow on the design lacks a background. It is crucial to address this issue to ensure consistency with the overall design aesthetic. To fix this, you can use `background: none;` to remove the background from the button. This will give the button a cleaner and more polished look, which will improve the user's overall experience.

Reply:
Dear David: Your solution (background: none) doesn't work. It's not enough to eliminate the gray background color that the arrow buttons were showing, eventhough the images didn't have background (they were transparent in Gimp).